src/status_im/ui/screens/discover/all_dapps/views.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im.ui.screens.discover.all-dapps.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [cljs.pprint :as pprint]
+            [re-frame.core :as re-frame]
+            [status-im.components.react :as react]
+            [status-im.components.chat-icon.screen :as chat-icon]
+            [status-im.components.carousel.carousel :as carousel]
+            [status-im.ui.screens.discover.components.views :as discover-components]
+            [status-im.ui.screens.discover.styles :as styles]
+            [status-im.utils.platform :as platform]
+            [status-im.i18n :as i18n]
+            [status-im.components.toolbar-new.view :as toolbar]))
+
+(defn dapp-item [dapp]
+  (let [{:keys [name photo-path dapp?]} dapp
+        ;; TODO(oskarth): Refactor this to be in discover styles, just uppercase
+        item-style (get-in platform/platform-specific [:component-styles :discover :item])]
+    [react/view
+     [react/view styles/dapps-list-item
+      [react/view styles/dapps-list-item-second-row
+      [react/view styles/dapps-list-item-name-container
+       [react/view (merge styles/dapps-list-item-avatar-container
+                          (:icon item-style))
+        ;; TODO(oskarth): Get rid of badge on icon
+        [react/view (chat-icon/contact-icon-contacts-tab dapp)]]
+       [react/text {:style           styles/dapps-list-item-name
+                    :font            :medium
+                    :number-of-lines 1}
+        name]]]]]))
+
+(defn preview [dapps]
```

Comment:
Can you move this one to `status-im.ui.screens.discover.views`? That way all code that appears on the same screen is in the same namespace. That was the criteria, not the function subject. It should make it easy to find the right code just by knowing the layout of the app.

On second thought, hold that one. It makes sense to do that in the PR for #1968

Reply:
Agree; will move in next PR

Reply:
Marked with TODO